content/blog/2024-01-16-deploying-mcg-with-cisco-flashstack-portworx.md

```diff
@@ -0,0 +1,28 @@
+
+At the core of this solution lies Cisco's FlashStack Data Center, a robust and scalable infrastructure designed to support diverse workload requirements. The hardware and software components employed in Cisco's internal labs validate the efficacy of this solution, ensuring a reliable and efficient deployment framework.
+
+In this white paper, Cisco outlines the hardware and software components and how they provide value for cloud-native workloads delivered across a hybrid cloud environment. The Cisco FlashStack solution provides an outstanding unified system to deliver robust, business critical applications. Red Hat's Multicloud GitOps Validated Pattern delivers a codified GitOps framework for deploying and configuring distributed software architectures in a predictable, repeatable, and extensible manner. By combining a ready-built system infrastructure with a validated deployment solution for Red Hat OpenShift, Portworx Enterprise, and other core software components, organizations can accelerate the time to value for a modern application development and deployment platform. Notably, the flexibility of this solution extends beyond the unified hardware infrastructure and incorporates a complete cloud-native application development and deployment platform with key management, security, registry, and cluster data management services.
+
+The deployment of [Red Hat OpenShift Container Platform](https://www.redhat.com/en/technologies/cloud-computing/openshift/container-platform) serves as a cornerstone of this solution. The Multicloud GitOps Validated Pattern seamlessly deploys additional solution components, such as [Red Hat Advanced Cluster Management](https://www.redhat.com/en/technologies/management/advanced-cluster-management), [Red Hat Advanced Cluster Security](https://www.redhat.com/en/technologies/cloud-computing/openshift/advanced-cluster-security-kubernetes), [Red Hat Quay](https://www.redhat.com/en/technologies/cloud-computing/quay), and[Portworx Enterprise](https://portworx.com/services/kubernetes-storage/) providing a complete platform. Leveraging the robustness of Cisco UCS server platforms, organizations can manage and orchestrate containerized applications on-premises and across multi cloud environments. The versatility of Red Hat OpenShift, coupled with the reliability of Cisco UCS servers, forms a powerful synergy driving efficiency and performance.
```

Comment:
MCGO doesn't deploy Quay, normally. Did Cisco include that?

```and[Portworx Enterprise]```
^ Need a space here